README.md

```diff
@@ -7,22 +7,47 @@ npm install vue-lazzy-image
 ## How use lib
 
 ```ecmascript 6
-import lazyload from 'vue-lazzy-image'
+import CoreLazyLoading from 'vue-lazzy-image'
 
 ...
 
-directives: {
-  lazyload,
+components: {
+  CoreLazyLoading,
 },
 ```
 
 ## Template
 
 ```ecmascript 6
-<img
-  v-lazyload
-  src="image_link"
-  class="class_name"
-  alt="alt_name"
-/>
+<CoreLazyLoading>
+  <img
+    src="image_link"
+    class="class_name"
+    alt="alt_name"
```

Comment:
For parameters should be created table that shows all props of our component. An example will be provided below:

[Readme with tables](https://github.com/ibodzoda-001/angular-osm/blob/main/README.md)

This README file could also be helpful:
View: [README View](https://raw.githubusercontent.com/blueimp/JavaScript-Templates/master/README.md)
Sour code: [README Source code](https://raw.githubusercontent.com/blueimp/JavaScript-Templates/master/README.md)